cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -781,6 +785,30 @@ instance ToObject (TraceLocalTxSubmissionServerEvent blk) where
   toObject _verb _ =
     mkObject [ "kind" .= String "TraceLocalTxSubmissionServerEvent" ]
 
-instance ToObject (Consensus.TraceForgeEvent blk) where
-  toObject _verb _ =
-    mkObject [ "kind" .= String "TraceForgeEvent" ]
+instance ToObject (TraceForgeEvent blk) where
+  toObject _verb (TraceForgeEvent slotNo _) =
+    mkObject
+        [ "kind" .= String "TraceForgeEvent"
+        , "slot" .= toJSON (unSlotNo slotNo)
+        ]
+  toObject _verb (TraceCouldNotForge slotNo _) =
+    mkObject
```

Comment:
there is more information in the second parameter that could be traced here. `AnachronyFailure`